activerecord/lib/active_record/schema_migration.rb

```diff
@@ -24,10 +24,9 @@ def table_exists?
         ActiveSupport::Deprecation.silence { connection.table_exists?(table_name) }
       end
 
-      def create_table(limit=nil)
+      def create_table
```

Comment:
Looks like that `SchemaMigration` is an internal class because this class is hidden from the doc.

https://github.com/rails/rails/blob/v5.0.0.beta1/activerecord/lib/active_record/schema_migration.rb#L9

Do we need to deprecation even if an internal class?